runbooks/k8s-nginx-ingress-runbook.md

```diff
@@ -0,0 +1,106 @@
+# UPP Kubernetes NGINX Ingress
+
+A helm deployment package for a simple NGINX ingress service via an AWS ELB.
+
+## Primary URL
+
+<https://github.com/Financial-Times/k8s-nginx-ingress>
+
+## Service Tier
+
+Platinum
+
+## Lifecycle Stage
+
+Production
+
+## Delivered By
+
+content
+
+## Supported By
+
+content
+
+## Known About By
+
+- donislav.belev
+- mihail.mihaylov
+- boyko.boykov
+- dimitar.terziev
+- miroslav.gatsanoga
+- kalin.arsov
+
+## Host Platform
+
+AWS
+
+## Architecture
+
+An Ingress Controller is a daemon, deployed as a Kubernetes pod, that watches the apiservers's /ingresses endpoint for updates to the Ingress resource. Its job is to satisfy requests for Ingresses.
+There are routing stacks in PAC, Publish and Delivery clusters that use NGINX Ingress Controllers from Kubernetes.
```

Comment:
Not sure if it's worth mentioning but as far as I can see in the Publishing and Delivery clusters Prometheus is the only service that uses Ingress.

Reply:
Yes, I think it is only Prometheus, but I decided to not be too specific since it is an architecture overview.